lib/iris/pandas.py

```diff
@@ -192,39 +134,42 @@ def as_data_frame(cube, copy=True):
 
     Kwargs:
 
-        * copy - Whether to make a copy of the data.
-                 Defaults to True. Must be True for masked data
-                 and some data types (see notes below).
```

Comment:
We can potentially bring this back.

Reply:
Agreed. Back in [332e260](https://github.com/SciTools/iris/pull/4669/commits/332e2606bbdfa1eed9e853f9028f94223483bc18)